metpy/tests/test_xarray.py

```diff
@@ -84,6 +84,13 @@ def test_units(test_var):
     assert test_var.metpy.units == units('kelvin')
 
 
+def test_units_percent():
+    """Test that '%' is converted to 'percent' """
```

Comment:
Just a few lint issues, need a period of the docstring.

```suggestion
    """Test that '%' is converted to 'percent'."""
```